src/static/clientConfig.ts

```diff
@@ -31,6 +31,7 @@ export interface ClientConfigInit<Params extends BaseUriParameters> {
     data: T,
     headers: {[key: string]: string}
   ) => Required<FetchOptions>['body'];
+  throwOnBadResponse?: boolean;
```

Comment:
There is a story to redo the README, but please add a description of this new parameter to to current one so it can easily be moved to the new one!